privacy-on-beam/pbeam/mean.go

```diff
@@ -89,8 +89,20 @@ type MeanParams struct {
 // values.  This aggregation is not hardened for such applications yet.
 //
 // MeanPerKey transforms a PrivatePCollection<K,V> into a PCollection<K,float64>.
-func MeanPerKey(s beam.Scope, pcol PrivatePCollection, params MeanParams) beam.PCollection {
+func MeanPerKey(s beam.Scope, pcol PrivatePCollection, params MeanParams, partitions ...beam.PCollection) beam.PCollection {
+	if len(partitions) > 1 {
+		log.Exitf("Only one partition PCollection can be specified. %v were specified.", len(partitions))
+	}
+
 	s = s.Scope("pbeam.MeanPerKey")
+
+	if len(partitions) == 1 {
+		if pcol.codec.KType.T != partitions[0].Type().Type() {
```

Comment:
Also here - can we assume the elements are all the same type?

Reply:
ah, never mind, this isn't checking the first partition's type, but the pcollection's setting. I misread